docs_app/content/guide/glossary-and-semantics.md

```diff
@@ -0,0 +1,136 @@
+
+### Multicast
+
+The act of one [producer](#Producer) being [observed](#Observation) by **many** [consumers](#Consumer).
+
+### Unicast
+
+The act of one [producer](#Producer) being [observed](#Observation) **only one** [consumer](#Consumer). An observable is "unicast" when it only connects one [producer](#Producer) to one [consumer](#consumer). Unicast doesn't necessarily mean ["cold"]([#3254](https://github.com/ReactiveX/rxjs/issues/3254)
+
+### Cold
+
+An observable is "cold" when it creates a new [producer](#Producer) during [subscribe](#Subscribe) for every new [subscription](#Subscription). As a result, a "cold" observables are _always_ [unicast](#Unicast), being one [producer](#Producer) [observed](#Observation) by one [consumer](#Consumer). Cold observables can be made [hot](#Hot) but not the other way around.
+
+### Hot
+
+An observable is "hot", when its [producer](#Producer) was created outside of the context of the [subscribe](#Subscribe) action. This means that the "hot" observable is almost always [multicast](#Multicast). It is possible that a "hot" observable is still _technically_ unicast, if it is engineered to only allow one [subscription](#Subscription) at a time, however, there is no straightforward mechanism for this in RxJS, and the scenario is a unlikely. For the purposes of discussion, all "hot" observables can be assumed to be [multicast](#Multicast). Hot observables cannot be made [cold](#Cold).
+
+## Minor Entities
+
+### Operator
+
+A factory function that creates an [operator function](#Operator_Function). Examples of this in rxjs are functions like [`map`](/api/operators/map) and [`mergeMap`](/api/operators/mergeMap), which are generally passed to [`pipe`](/api/index/class/Observable#pipe). The result of calling many operators, and passing their resulting [operator functions](#Operator_Function) into pipe on an observable [source](#Source) will be another [observable](#Observable), and will generally not result in [subscription](#Subscription).
+
+### Operator Function
+
+A function that takes an [observable](#Observable), and maps it to a new [observable](#Observable). Nothing more, nothing less. Operator functions are created by [operators](#Operator). If you were to call an rxjs operator like [map](/api/operators/map) and put the return value in a variable, the returned value would be an operator function.
+
+### Operation
+
+An action taken while handling a [notification](#Notification), as set up by an [operator](#Operator) and/or [operator function](#Operator_Function). In RxJS, a developer can chain several [operator functions](#Operator_Function) together by calling [operators](#Operator) and passing them to [pipe](/api/index/class/Observable#pipe), which results in a new [observable](#Observable). During [subscription](#Subscription) to that observable, operations are performed in an order dictated by the [observation chain](#Observation_Chain).
```

Comment:
"passing them to pipe": the phrasing is confusing: you don't pass them to pipe. You pass the operator functions that they have created. So maybe rephrase to

> In RxJS, a developer can chain several operator functions together by calling operators, and passing the created operator functions to pipe.